packages/ui/src/locales/ar-eg.json

```diff
@@ -497,6 +510,32 @@
   "shared-settings.edit.password.new": "كلمة السر الجديدة",
   "shared-settings.edit.password.confirm": "تاكيد كلمة السر الجديدة",
   "shared-settings.edit.password.confirm.not-same": "الرقم السري غير متطابق",
+  "settings.edit.personal.topics.loading": "برجاء الانتظار... جاري تحميل المفضلات!",
+  "settings.edit.personal.topics.error": "عذرًا، حدث خطأ أثناء تحميل المفضلات. برجاء المحاولة مرة أخرى!",
+  "settings.edit.personal.topics.title": "إضافة المفضلات",
+  "settings.edit.personal.topics": "قم باختيار المجالات التي تريد التحدث عنها",
+  "settings.edit.password.title": "تغيير كلمة السر",
+  "settings.edit.password.current": "كلمة السر الحالية",
+  "settings.edit.password.new": "كلمة السر الجديدة",
+  "settings.edit.password.confirm": "تاكيد كلمة السر الجديدة",
+  "settings.edit.password.confirm.not-same": "الرقم السري غير متطابق",
+  "settings.notifications": "الإشعارات",
+  "settings.notifications.lesson-date.title": "إشعارات تاريخ الدرس",
+  "settings.notifications.lesson-date.description": "سيتم اشعارك بميعاد الدرس قبله بيوم و قبله بساعه",
+  "settings.notifications.messages.title": "إشعارات الرسائل",
+  "settings.notifications.messages.description": "سوف يتم اشعارك بجميع الرسائل الواردة",
+  "settings.notifications.coming-soon.title": "قريبا 🎉",
+  "settings.notifications.coming-soon.description": "بإمكانك التحكم في الاشعارات بمجرد ان يتم تفعيل هذه الخاصية.",
+  "settings.topics.placeholder": "الرياضة، السفر، التكنولوجيا",
+  "settings.verify.title": "تأكيد الحساب الخاص بك.",
+  "settings.verify.description": "أكد حسابك حتي تتمكن من حجز الحصص او التحدث مع المعلمين، تفقد بريدك الإلكتروني لتجد رابط التاكيد.",
```

Comment:
I believe these are duplicates. Student related ids should be prefixed with `student-settings`. Please check the latest master. @mostafakamar2308